ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -83,23 +127,42 @@ export default function EditGasDisplay({
             {t('gasDisplayAcknowledgeDappButtonText')}
           </Button>
         )}
-
+        {isGasTooLow && (
+          <div className="edit-gas-display__error">
+            <Typography
+              color={COLORS.ERROR1}
+              variant={TYPOGRAPHY.H7}
+              align={TEXT_ALIGN.CENTER}
+            >
+              {t('editGasTooLow')}
+            </Typography>
+          </div>
+        )}
         {!requireDappAcknowledgement && (
           <RadioGroup
             name="gas-recommendation"
             options={[
-              { value: 'low', label: t('editGasLow'), recommended: false },
+              {
+                value: 'low',
+                label: t('editGasLow'),
+                recommended: defaultEstimateToUse === 'low',
+              },
               {
                 value: 'medium',
                 label: t('editGasMedium'),
-                recommended: false,
+                recommended: defaultEstimateToUse === 'medium',
```

Comment:
I see 'low', 'medium' and 'high' strings used in multiple places. Let's move it into constants instead?